### ISSUE TITLE
[beta] steam chat window is significantly CPU intensive, even idle and hidden

### ISSUE THREAD

**Issue by sylware** [ValveSoftware/steam-for-linux#7241]
#### Your system information

* Steam client version (build number or date): 
* Distribution (e.g. Ubuntu): 
* Opted into Steam client beta?: [Yes/No]
* Have you checked for system updates?: [Yes/No]

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

#### Steps for reproducing this issue:

1. 
2. 
3.

**Comment by dearlygparted** [comment]
It's happening on the stable client as well. Checking htop, I'm seeing the same 20-30% CPU usage by various Steam processes, looking at htop. ~/.local/share/Steam/ubuntu12_32/steam and ~/.local/share/Steam/ubuntu12_64/steamwebhelper are the main culprits. Seconds after closing the chat window, CPU usage goes back down to normal.

**Comment by olekolek1000** [comment]
Steam on almost all graphics cards does not use hardware accelerated rendering. Avatar glitter animation causes that.

**Comment by parkerlreed** [comment]
I wonder, are the new animated avatar backgrounds always playing? I'm also seeing extremely high usage with just the friends list and chat windows open.

![image](https://user-images.githubusercontent.com/841440/88335509-6ddbbc80-cd01-11ea-9a42-22590d0f78e2.png)

![image](https://user-images.githubusercontent.com/841440/88335510-703e1680-cd01-11ea-957f-a8f8ee54f9a3.png)

**Comment by parkerlreed** [comment]
GPU rendering for web views on or off makes no difference. 

Steam specs https://gist.github.com/parkerlreed/17e2478499ae43934bc88c8e3aae432f

**Comment by dearlygparted** [comment]
Seems like this issue may not be unique to Steam for Linux:

https://steamcommunity.com/discussions/forum/11/2798376099463199982/

**Comment by tanjilk** [comment]
Even here, even more so for the golden profile, which makes it worse

**Comment by vintagepc** [comment]
Yes, I am seeing this too with any window that has animated avatar frames. 

PLEASE give us an option to turn them off!

**Comment by sylware** [comment]
my old system did melt, literally. My new system is too beefy to show the cpu eating. I noticed some new options appearing in the steam chat client configuration related to this issue.

**Comment by snoazll** [comment]
I can confirm that this is still the case today. I didn't realise this until [this comment](https://github.com/ValveSoftware/steam-for-linux/issues/7372#issuecomment-697386828) in #7372 but this was why I was having Steam using around %20-30 of CPU when idling in the background. As soon as I went offline on Steam Friends, the CPU usage went back to normal.

**Comment by broconut123** [comment]
Been having some major fps drops in Dota 2 on linux when tabbing in and out to write messages to friends through steam. Looking at the process the steam process has very high CPU usage resulting in dota going from 100+ fps to >60 fps.
The problem goes away when shutting down the steam windows.

**Comment by f0o** [comment]
I temporarily resolved it by turning off all chat animations (avatars, backgrounds, whatnots), worth a shot

**Comment by zatricky** [comment]
Strong chance this and #6630 are related. A video I added there also shows how bad the performance issue can be.

**Comment by sylware** [comment]
On my zen2: turning off the chat animation does spare off one ~1% cpu steamwebhelper, another ~0.2% cpu steamwebhelper and the 32bits steam process at ~0.3% cpu. An heavily animated profile costs one ~3% cpu steamwebhelper + another ~1% cpu steamwebhelper. If I select the library, I am back at 0%.

The teaching is: up to ~%5 cpu at doing nothing on beefy specs hardware, then on medium/low specs hardware... always have the library selected (or minimal mode, if still around) and do switch off chat.

**Comment by randompast** [comment]
Linux Mint 21, fresh install, 30% utilization across all cores on a 7950x.  Toggling steam chat on and off multiple times changes the utilization accordingly.  This is unreasonable.
